SQLite.CodeFirst/Builder/CreateIndexStatementBuilder.cs

```diff
@@ -54,9 +54,9 @@ public CreateIndexStatementCollection BuildStatement()
             return new CreateIndexStatementCollection(createIndexStatments.Values);
         }
 
-        private static string GetIndexName(IndexAttribute index, EdmProperty property)
+        private string GetIndexName(IndexAttribute index, EdmProperty property)
         {
-            return index.Name ?? "IX_" + property.Name;
+            return index.Name ?? string.Format("IX_{0}_{1}", entitySet.Table, property.Name);
```

Comment:
Thanks for your PR.
There is an extension Method "GetTableName()"  `SQLite.CodeFirst.Extensions.EntityTypeExtension`. Can you use this method to get the table Name instead of `entitySet.Table`?